data/categories.txt

```diff
@@ -1810,17 +1810,30 @@ sw:^Reli ya chini
 railway-level_crossing
 en:^Level Crossing
 ru:^Железнодорожный переезд
+uk:^Level Crossing
```

Comment:
Note that the "^" sign is not required here as there is no ambiguity.

Reply:
I know. However, many other entries in categories.txt use a redundant "^"-sign, too, so I decided to stick with this kind of notation.